manifests/init.pp

```diff
@@ -0,0 +1,42 @@
+# == Class: puppetcrl_sync
+#
+# A Puppet module for distributing the Puppet CA CRL to other hosts
+#
+# === Parameters
+#
+# [*crl_location*]
+#   String.  Full path to the puppet CA CRL file
+# [*crl_target*]
+#   String.  Full path to the target location for the CRL file
+# [*crl_owner*]
+#   String. Owner for the CRL file in the target location
+# [*crl_group*]
+#   String. Group for the CRL file in the target location
+# [*crl_mode*]
+#   Integer. Octal mode for the CRL file in the target location
+#
+#
+class puppetcrl_sync (
+  $crl_location = $puppetcrl_sync::params::crl_location,
```

Comment:
Again up to you but prefixing everything with crl seems a bit superfluous as there are no name space clashes.